kbase-extension/static/kbase/js/widgets/appWidgets2/input/select2ObjectInput.js

```diff
@@ -195,6 +196,10 @@ define([
                         return Validation.validateWorkspaceObjectName(processedValue, validationOptions);
                 }
             });
+            // Need to create catch for promise!!!!
```

Comment:
There should be a catch for this promise that gets bubbled up, so any errors don't get swallowed.